Documentation/Doxygen/MainPage.dox

```diff
@@ -2,7 +2,7 @@
  *
  * \mainpage  Insight Toolkit
  *
- * \image html itkLogo.png
+ * \image html itkLogo.png width=50% height=50%
```

Comment:
Is this 50% of the image's original size, or 50% of screen/page size?

Reply:
This is 50% of the image's original size. The image size is relative to the original size of the image itself, not the screen or page size.

Thank you for reviewing!